members/CAL000051.yaml

```diff
@@ -43,7 +43,7 @@ contact_form:
         - name: accent-replace
           values: ["$NAME_FIRST", "$NAME_LAST"]
           selectors: ["#txtFirstName", "#txtLastName"]
-          commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/[èēé]/g,'e').replace(/[àå]/g, 'a').replace(/[Á]/g, 'A').replace(/[.&,()@]/g, '').replace(/[ý]/g, 'y').replace(/[í]/g, 'i').replace(/[ó]/g, 'o'); }"]
+          commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/[èēé]/g,'e').replace(/[àåáä]/g, 'a').replace(/[Á]/g, 'A').replace(/[.&,]/g, '').replace(/[ý]/g, 'y').replace(/[íï]/g, 'i').replace(/[óö]/g, 'o'); }"]
```

Comment:
Removed the `()`?

Reply:
I could add it back, but thought of it as minor since i haven't really seen any instances of that lately

Reply:
I guess I don't care if it was intentional, seems like it was harmless to leave it 🤷‍♂